### PR TITLE
Added option in general settings for fullscreen view submissions links

### PR DESCRIPTION
**Pull request by rosenpin**

### CODE REVIEW

File: app/src/main/java/me/ccrama/redditslide/Activities/FullscreenImage.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
 /**
  * Created by ccrama on 3/5/2015.
  */
-public class FullscreenImage extends BaseActivity {
+public class FullscreenImage extends FullScreenActivity {
```
Comment (l3d00m) [inline app/src/main/java/me/ccrama/redditslide/Activities/FullscreenImage.java]: Why extending it from a new class when you already checked for it in line 51-54?

**Comment by l3d00m** [comment]
this includes #679

**Comment by ccrama** [comment]
Thank you so much for you guys' work! Really sleek! Thanks

File: app/src/main/java/me/ccrama/redditslide/Activities/FullscreenImage.java (hunk 1 above)
Comment (rosenpin) [inline app/src/main/java/me/ccrama/redditslide/Activities/FullscreenImage.java]: So it won't have to be written in every full screen activity but in the full screen activities only

**Pull request merged**